### PR TITLE
saved to pinned contracts transition

### PR DESCRIPTION
**Pull request by Aniket-Engg**

### CODE REVIEW

File: libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx [hunk 1]
```diff
@@ -291,7 +321,7 @@ export function UniversalDappUI(props: UdappProps) {
           { props.isSavedContract && props.instance.filePath ? (
             <div className="d-flex" data-id="instanceContractFilePath">
               <label>
-                <b><FormattedMessage id="udapp.filePath" />:</b> {props.instance.filePath}
+                <span style={{float: 'left'}}><b><FormattedMessage id="udapp.filePath" />:</b></span> &nbsp;{props.instance.filePath}
```
Comment (ioedeveloper) [inline libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx:324]: Using a class here for float will be better and also margin/padding in place of `'&nbsp;'`

Comment (Aniket-Engg) [reply libs/remix-ui/run-tab/src/lib/components/universalDappUI.tsx:324]: I will remove that as of now, as in next PR we can remove this field

File: apps/remix-ide/src/app/tabs/locales/en/udapp.json [hunk 2]
```diff
@@ -65,16 +65,18 @@
   "udapp.tooltipText3": "Click to open a bridge for converting L1 mainnet ETH to the selected network currency.",
 
   "udapp._comment_instanceContainerUI.tsx": "libs/remix-ui/run-tab/src/lib/components/instanceContainerUI.tsx",
-  "udapp.deployedContracts": "Deployed Contracts",
+  "udapp.deployedContracts": "Deployed/Unpinned Contracts",
   "udapp.deployAndRunClearInstances": "Clear instances list and reset recorder",
-  "udapp.deployAndRunNoInstanceText": "Currently you have no deployed contracts to interact with.",
-  "udapp.tooltipText6": "Autogenerated generic user interfaces for interaction with deployed contracts",
+  "udapp.deployAndRunNoInstanceText": "Currently you have no unpinned contracts to interact with.",
```
Comment (ioedeveloper) [inline apps/remix-ide/src/app/tabs/locales/en/udapp.json:70]: I think removing "unpinned" here will better simplify the message. "Currently you have no contracts to interact with."

File: apps/remix-ide/src/app/tabs/locales/en/udapp.json [hunk 3]
```diff
@@ -65,16 +65,18 @@
...
+  "udapp.tooltipText6": "Autogenerated generic user interfaces for interaction with deployed/unpinned contracts",
 
-  "udapp.savedContracts": "Saved Contracts",
-  "udapp.NoSavedInstanceText": "No saved contracts found.",
-  "udapp.tooltipTextUnsave": "Unsave & move to Deployed Contracts list",
-  "udapp.savedOn": "Saved On",
-  "udapp.filePath": "File Path",
+  "udapp.savedContracts": "Pinned Contracts",
+  "udapp.tooltipTextPinnedContracts": "List of pinned contracts for selected network",
+  "udapp.NoSavedInstanceText": "No contracts pinned yet. Pinned contracts stay after reloading Remix IDE.",
```
Comment (ioedeveloper) [inline apps/remix-ide/src/app/tabs/locales/en/udapp.json:75]: Also, i think this info will be better in a bootstrap alert. Something like the attached screenshot.
Screenshot 2024-03-08 at 11 45 21

Comment (Aniket-Engg) [reply apps/remix-ide/src/app/tabs/locales/en/udapp.json:75]: That can be discussed and come in a different PR

Comment (Aniket-Engg) [reply apps/remix-ide/src/app/tabs/locales/en/udapp.json:75]: Liana wanted it as info instead of warning

Comment (ioedeveloper) [reply apps/remix-ide/src/app/tabs/locales/en/udapp.json:75]: Yes, we could have it as a bootstrap alert info

**Review by ioedeveloper — APPROVED** [review]

**Review by joeizang — APPROVED** [review]

**Pull request merged**